src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -496,7 +496,18 @@ else if (packet instanceof RtcpSrPacket)
         {
             // Allow the BC to update the timestamp (in place).
             RtcpSrPacket rtcpSrPacket = (RtcpSrPacket) packet;
-            bitrateController.transformRtcp(rtcpSrPacket);
+            if (!bitrateController.transformRtcp(rtcpSrPacket))
+            {
+                if (logger.isDebugEnabled())
+                {
+                    logger.debug(
+                        "dropping an sr from ssrc="
+                            + rtcpSrPacket.getSenderSsrc()
+                            + ", timestamp="
+                            + rtcpSrPacket.getSenderInfo().getRtpTimestamp());
+                }
+                return;
```

Comment:
Doesn't this drop all SRs for audio?

Reply:
Note that we're already filtering out the SRs for video streams that the BC doesn't accept (in `Endpoint.wants()`/`BitrateController.accept()`.

Reply:
right, i forgot that, i'll revert this